projects/js-packages/components/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
since we add new files to the package (and later on to the Jetpack plugin itself), we'll need to ensure the files aren't shipped with the production version, by adding a reference to the `tsconfig.json` files in each `.gitattributes` file.

Reply:
Thanks, will do